### ISSUE TITLE
PhyreDummyShaderCreator

### ISSUE THREAD

**Issue by quanpn90** [Trails-Research-Group/Doc#2]
Hello! Thank you very much for writing the guide. 

I am trying to follow the adding models (from CS2, I guess the principle is similar than Kuro) to CS4 guide and having troubles at the shaders part. If I understood correctly, what we can do is to extract the shader parameters from the models, and then find the shaders with similar parameters compiled in the game to reuse, is that correct?

It was pretty hard for me to move on because I don't know how to convert ed8.fx#....phyre to ed8.fx file.

Could I ask for sharing the PhyreDummyShaderCreator software? 
Even if I couldn't port the models (from CS2), it still helps a lot in analyzing the parameters of the shaders.

Best regards,
Quan

**Comment by TwnKey** [comment]
Hi,
shaders are the reason why I burnt out and gave up on that guide (also the reason why I didnt end up making it work on my end).
But I will try to share the tools later if I can.

There is only one person i know who managed to make successful model ports with multiple shaders and he doesn't want to go public (author of shizuna mod)

**Comment by quanpn90** [comment]
Hi, thanks for your answer!

I can wait until you recover from burning out. Personally I have some models with 1 or 2 shaders that are probably easier to practice, I don't have much experience with maya/modeling but I really want to try.

**Comment by TwnKey** [comment]
I have updated the guide with the latest changes I got from *the one who shall not be named* regarding the shaders, it helps to set them up, but the process is still stupidly hard right now.
Anyway, you will find the tools in the WorkFolder that is linked in the guide, environment section, including the asset processor and the dummy shader creator.

I can't help you unfortunately as I never managed to make it work (I did do Shrek back in the day with a single shader used as Default Shader and specified in PhyreAssetSpec.xml, but beyond that I can offer no help.

Thus, good luck with that. I'll close the issue if you don't mind as the tools you asked for were provided.
